docs/20_smart-contracts/30_tutorials/20_create-an-nft.md

```diff
@@ -0,0 +1,769 @@
+---
+title: Create an NFT
+---
+
+## Want to just create an NFT easily?
+
+In this tutorial we are going to discuss creating an NFT that follows Ethereum's ERC721
+standard so that we can dig into some technicals using a clear standard. 
+
+**However**, if you want to create an NFT that follows the [**Atomic Assets**](https://github.com/pinknetworkx/atomicassets-contract) standard which
+is more common on the EOS Network, you can visit the [Atomic Assets NFT Creator](https://eos.atomichub.io/creator)
+where you can easily create an NFT that will instantly be listed on the AtomicHub marketplace.
+
+## What is an NFT?
+
+An NFT is a **non-fungible token**. This means that it is a unique token that cannot be
+interchanged with another token. 
+
+Let's take a trading card as an example. If you have a trading card of a blue car, 
+you can't trade it for a trading card of a red car. They are different cards and 
+therefore are not _fungible_.
+
+## What is an NFT Standard?
+
+An NFT standard is a set of rules that all NFTs must follow. This allows for NFTs to be
+interoperable with other NFTs and for applications like marketplaces and wallets to
+understand how to interact with them.
+
+## What is the ERC721 Standard?
+
+The [ERC721 standard](https://eips.ethereum.org/EIPS/eip-721) is an NFT standard that was created by the Ethereum community. It
+is the most common NFT standard and is used by many NFTs on the Ethereum network. If you've
+ever seen a Bored Ape, they are ERC721 NFTs.
+
+![Bored Ape Club Examples](./images/boredapeclub.jpg)
```

Comment:
I'd rather stick with it. It's the most notable NFT in existence, which has been on television, movies, and plastered across the internet. If you've heard of NFTs, you've heard of the Bored Ape Club. 

We also can't use an Atomic Assets NFT or any other EOS ecosystem NFT for this guide because it's about mimicking the ERC721 standard on EOS.